### PR TITLE
Release on github tagging

### PR DESCRIPTION
**Pull request by skjnldsv**
@ChristophWurst how did you released the last one, you forgot to push the github tags?

### CODE REVIEW

**Comment by ChristophWurst** [comment]
> @ChristophWurst how did you released the last one, you forgot to push the github tags?

Locally, I guess :)

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+      - name: Install dependencies & build
+        run: |
+          npm ci
+          npm run build --if-present
+
+      - name: Publish to npm
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```
Comment (ChristophWurst) [inline .github/workflows/npm-publish.yml]: how can you set this?

Comment (skjnldsv) [reply .github/workflows/npm-publish.yml]: It's set in github's repository settings :)

**Comment by skjnldsv** [comment]
> Locally, I guess :)

Remember to update github as well, can you push the unpushed tag? :)

File: .github/workflows/npm-publish.yml (hunk 1 above)
Comment (skjnldsv) [reply .github/workflows/npm-publish.yml]: https://github.com/nextcloud/eslint-config/settings

Comment (skjnldsv) [reply .github/workflows/npm-publish.yml]: (same goes for @nextcloud/vue)

**Comment by skjnldsv** [comment]
> can you push the unpushed tag? :)

I just did it :)
https://github.com/nextcloud/eslint-config/releases/tag/v1.0.0

It was actually @korelstar  :p

**Comment by korelstar** [comment]
Sorry, I wasn't aware that I created a tag with that commit. I just changed the package name and version (manually, not with `npm version`). I think the tag was created automatically. :see_no_evil:

**Comment by skjnldsv** [comment]
Yes the tag is doen auto, but you forgot to push it :)
It's ok, don't worry ;)

**Pull request merged**